src/views/studio/EditVideoSheet/EditVideoSheet.tsx

```diff
@@ -203,18 +198,24 @@ export const EditVideoSheet: React.FC = () => {
     }
 
     const refetchDataAndCacheAssets = async (videoId: VideoId) => {
-      const fetchedVideo = await refetchVideo({ where: { id: videoId } })
-
+      const fetchedVideo = await client.query<GetVideosConnectionQuery, GetVideosConnectionQueryVariables>({
+        query: GetVideosConnectionDocument,
+        variables: {
+          where: {
+            channelId_eq: activeChannelId,
+            isPublic_eq: data.isPublic,
+            id_eq: videoId,
+          },
+        },
+      })
```

Comment:
Why do we use a connection here instead of a regular `videoByUniqueInput` query? We care only about one video details, right?

Reply:
I think it works and I'll change that, but what about the `cursor` field? I cannot add a video cursor to the cache when I do the `videoByUniqueInput` query. Do we need that in this case?

Reply:
Noticed one more thing - when I'm using `videoByUniqueInput` after adding video it refetches the whole `videosConnection` list. When I'm using `videosConnection` it doesn't. To be honest, I'm not sure why. 

Also,  I need to do something with `totalCount: existing.totalCount + 1`, because analogous to `removeVideoFromCache`, `totalCount` is incremented in unlisted, public and all videos query.

Reply:
> Noticed one more thing - when I'm using videoByUniqueInput after adding video it refetches the whole videosConnection list.

Hmm that's weird.

Maybe let's use `videosConnection` after all, it will give us a cursor as well. However, if we want to get the same cursor as other queries, we will need to add `orderBy` here, it changes the cursor.

Reply:
Regarding totalCount I'm not sure... it's messy. I think once we get a normalized cache, we can just return `totalCount` locally instead

Reply:
Does it break things a lot right now or can we ignore it for the time being?

Reply:
>Maybe let's use videosConnection after all, it will give us a cursor as well. However, if we want to get the same cursor as other queries, we will need to add orderBy here, it changes the cursor.

So, I'll revert that.

> Does it break things a lot right now or can we ignore it for the time being?

Not really. I think we can ignore it for now.